x-pack/test/saved_object_tagging/api_integration/tagging_api/apis/create.ts

```diff
@@ -74,7 +74,7 @@ export default function ({ getService }: FtrProviderContext) {
               valid: false,
               warnings: [],
               errors: {
-                name: 'Tag name can only include a-z, 0-9, _, -,:.',
+                name: 'Tag name must be at least 2 characters',
```

Comment:
NIT: Should we add some functional tests to validate it works with special chars? Especially the search use case?